Pod/Classes/RZMessagingWindow.m

```diff
@@ -292,28 +292,48 @@ @implementation RZRootMessagingViewController
 
 #pragma mark - Orientation method overrides
 
--(BOOL)shouldAutorotate
+-(UIStatusBarStyle)preferredStatusBarStyle
 {
     UIViewController *topViewController = [RZRootMessagingViewController topViewController];
+
+    UIStatusBarStyle statusBarStyle;
     
-    if ( topViewController == self ) {
-        return [super shouldAutorotate];
+    if ( [(RZMessagingWindow *)self.view.window errorIsBeingPresented] ) {
+        statusBarStyle = [super preferredStatusBarStyle];
     }
     else {
-        return topViewController.shouldAutorotate;
+        statusBarStyle = topViewController.preferredStatusBarStyle;
```

Comment:
I don't think the style guide says anything about using dot notation for pseudo-properties. Feel free to submit a PR if you think it should, but I'm not hugely bothered by it. I know some other people are, though.